### PR TITLE
Change domain user model

### PR DESCRIPTION
**Pull request by DavidHeymann**
Change model and return value of domainUser

### CODE REVIEW

File: src/domainUser/domainUser.interface.ts [hunk 1]
```diff
@@ -6,5 +6,6 @@ export interface IDomainUser {
   domain: string;
   name: string;
   fullString?: string;
+  UID?: string;
```
Comment (eladex) [inline src/domainUser/domainUser.interface.ts]: adfsUID
adfsUid
adfsUId

File: src/person/person.controller.ts [hunk 2]
```diff
@@ -8,21 +8,24 @@ import { userFromString } from '../domainUser/domainUser.utils';
 import { DomainUserController } from '../domainUser/domainUser.controller';
 import { IDomainUser } from '../domainUser/domainUser.interface';
 import * as utils from '../utils.js';
-
+import { domainMatch } from 'tough-cookie';
```
Comment (eladex) [inline src/person/person.controller.ts]: WTF?

Comment (DavidHeymann) [reply src/person/person.controller.ts]: Done

File: src/domainUser/domainUser.interface.ts (hunk 1 above)
Comment (DavidHeymann) [reply src/domainUser/domainUser.interface.ts]: Done and chenge fullstring field to uniqueID

**Pull request closed without merging**